tokens-studio/figma-plugin
# Fix duplicating token group updating hanging

### Why does this PR exist?

Fixes #2755 



### What does this pull request do?

- [x] Removes redundant `await` for `duplicateTokenGroup` Redux action, so `completeJob` runs.



### Testing this change

To reproduce:

1. Create a token with a group e.g. foo/bar.
2. Right click the group `foo`
3. Go to Duplicate
4. Press submit
5. Verify that there is no loading bar and that the duplicate group has been created.



### Additional Notes (if any)

| Before | After |
|--------|-------|
| ![Before](https://github.com/tokens-studio/figma-plugin/assets/6757532/77d498c5-c0e6-4d35-9c4c-ec2297cdcd69) | ![After – Tokens / Typography / New Token](https://github.com/tokens-studio/figma-plugin/assets/6757532/86a56170-e6b6-4644-9230-9e51475dfe93) |
